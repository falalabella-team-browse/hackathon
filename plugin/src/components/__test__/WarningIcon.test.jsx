import React from "react";
import { shallow } from "enzyme";
import WarningIcon from "../WarningIcon";

describe("WarningIconIcon", () => {
  it("should render properly", () => {
    const wrapper = shallow(<WarningIcon />);
    expect(wrapper).toMatchSnapshot();
  });
});
