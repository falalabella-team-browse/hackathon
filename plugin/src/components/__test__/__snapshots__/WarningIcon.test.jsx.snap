// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WarningIconIcon should render properly 1`] = `ShallowWrapper {}`;
